e2e/scripts/st_heading.py

```diff
@@ -16,3 +16,14 @@
 
 st.header("This header is awesome!")
 st.header("This header is awesome too!", anchor="awesome-header")
+
+st.title("`Code` - Title with Anchor")
+st.title("`Code` - Title with Anchor", anchor="title")
+
+
+st.subheader("`Code` - Subheader with Anchor")
+st.subheader(
```

Comment:
I think the spaces in front of the second line are not planned, since it is rendered as a code block this way:
<img width="728" alt="image" src="https://user-images.githubusercontent.com/2852129/182340732-8fc1b9c3-d5d5-4f35-9bf8-e68088dd32e9.png">

Maybe remove those spaces:
```python
st.subheader(
    """`Code` - Subheader with Anchor
[link](href)""",
    anchor="subheader",
)
```